README.md

```diff
@@ -20,50 +20,58 @@ You can explore the following operations for Office 365:
 
 **Events**
 
-* Get the signed-in user's events from Office 365
-* Create a new event for the user
-* Update a user's event
-* Delete a user's event
+* [Get the signed-in user's events from Office 365.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/EventsSnippets.java#L53)
+* [Create a new event for the user.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/EventsSnippets.java#L68)
+* [Update a user's event.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/EventsSnippets.java#L94)
+* [Delete a user's event.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/EventsSnippets.java#L145)
 
 **Messages**
 
-* Get user's messages from Office 365.
-* Send a message from the user.
+* [Get user's messages from Office 365.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/MessageSnippets.java#L46)
+* [Send a message from the user.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/MessageSnippets.java#L58)
 
 **Contacts**
 
-* Get all contacts in the tenant.
-* Create a new contact in your tenant.
+* [Get all contacts in the tenant.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/ContactsSnippets.java#L37)
+* [Create a new contact in your tenant.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/UsersSnippets.java#L68)
 
 **Groups**
 
-* Get all groups in your tenant's directory.
-* Create a new security group.
-* Get information about a specific group in the tenant.
-* Update the description of a group.
-* Delete a group.
-* Get a group's members.
-* Get a group's owners.
+* [Get all groups in your tenant's directory.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/GroupsSnippets.java#L138)
+* [Create a new security group.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/GroupsSnippets.java#L152)
+* [Get information about a specific group in the tenant.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/GroupsSnippets.java#L57)
+* [Update the description of a group.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/GroupsSnippets.java#L170)
+* [Delete a group.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/GroupsSnippets.java#L197)
+* [Get a group's members.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/GroupsSnippets.java#L93)
+* [Get a group's owners.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/GroupsSnippets.java#L122)
 
 **Drives**
 
-* Gets the signed-in user's drive.
-* Gets  all of the drives in your tenant.
+* [Gets the signed-in user's drive.](/app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/DrivesSnippets.java#L55)
+* [Gets  all of the drives in your tenant.](/app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/DrivesSnippets.java#L68)
```

Comment:
I think there are several of these that don't link to the top of the snippet, so I'll stop commenting on them here. :-) I would just say they should all link to the name of the snippet, like get_organization_drives and such.